scss/animation.scss

```diff
@@ -0,0 +1,46 @@
+@keyframes moveToRight{
+    0%{
+        opacity: 0;
+        transform: translateX(-20%);
+    }
+    100%{
+        transform: translateX(0%);
+        opacity: 1;
+    }
+}
+
+@keyframes moveToLeft{
+    0%{
+        opacity: 0;
+        transform: translateX(20%);
+    }
+    100%{
+        transform: translateX(0%);
+        opacity: 1;
+    }
+}
+
+@keyframes moveDown {
+    0%{
+        transform: translateY(-20%);
+        opacity: 0;
+
+    }
+    100%{
+        transform: translateY(0);
+        opacity: 1;
+    }
+}
+
+@keyframes moveUp {
+    0%{
+        transform: translateY(0);
+        opacity: 0;
```

Comment:
we default opacity 0 for the parent class, add make it appear after animation, we don't set it in 0%, it will cause some problems.